packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -118,12 +118,31 @@ export class MediaPlaceholder extends Component {
 			onError,
 			onSelect,
 			value = [],
+			dynamicAlteration,
```

Comment:
Can we explain a bit more what this PR does? Is it needed? should we just always use that behavior?

Reply:
It is currently only in place as a safety measure to ensure this code is only running for the Gallery block.  I did not want to immediately throw it into use in all cases as I may be missing a use of it somewhere and did not want to accidentally break anything that I haven't been able to explicitly test.  It may not be needed in the end...

Reply:
I'd rather personally be more conservative about introducing new APIs. Each time we introduce something new (like this prop) we're committing to ensure backward compatibility for this prop. (if it's still in the plugin, it's two releases but once it can be forever).

So if we really think, this should be the default behavior, we should just avoid the prop.

Reply:
Great points!

I have removed the `dynamicAlteration` prop.  After searching through the codebase once more for all uses of the `MediaPlaceholder` component, it seems that the `Gallery` component is the only use of the `MediaPlaceholder` where the `multiple` property used (If this is not the case I would love to hear where else it is used, I have not been able to find anything).  So even without my unnecessary prop, it seems that the `Gallery` block is still the only block effected by this change.

Reply:
Thanks